functions/lambdas_param.vale

```diff
@@ -1,10 +1,14 @@
 // a lambda as a function parameter
+
+import stdlib.*;
+
 fn main() export {
   a = do({ 42 });
   println(a); // => 42
 
   b! = 10;
-  { mut b = 42; }(); // the lambda which is called changes b to 42
```

Comment:
This line revealed a bug (270) in the compiler, so I conveniently used an intermediate variable to work around this bug for now.